src/Core/Impl/Extensions/TaskCompletionSourceExtensions.cs

```diff
@@ -18,8 +18,10 @@
 
 namespace Microsoft.Python.Core {
     public static class TaskCompletionSourceExtensions {
+        private const int NO_DELAY = -1;
```

Comment:
AFAIK, we don't use all caps notation for constants.

Reply:
@AlexanderSher how should I change it? no_delay then?

Reply:
changed to lower case.